docs/gui/workorder-details.rst

```diff
@@ -9,8 +9,7 @@ This tool allows you to adjust item types, materials, and/or traits for items
 used in manager workorders. The jobs created from those workorders will inherit
 the details.
```

Comment:
the introductory text could use a few more examples of things you can do with this tool that you wouldn't be able to do in vanilla. what do you think are the most useful situations where this tool helps?

Reply:
probably use dyed or dyeable (=undyed I think) cloth, or encrust unimproved items...

I'm not quite sure.